planner/core/task.go

```diff
@@ -431,20 +454,36 @@ func (p *PhysicalHashJoin) GetCost(lCnt, rCnt float64) float64 {
 	// Cost of quering hash table is cheap actually, so we just compute the cost of
```

Comment:
```suggestion
	// Cost of querying hash table is cheap actually, so we just compute the cost of
```
Fix this typo by hand in this PR?